lottie/src/main/java/com/airbnb/lottie/network/DefaultFetcher.java

```diff
@@ -0,0 +1,54 @@
+package com.airbnb.lottie.network;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import androidx.annotation.NonNull;
+
+public class DefaultFetcher implements Fetcher {
```

Comment:
This would become `DefaultLottieNetworkFetcher`

Reply:
done